src/utils/createStore.js

```diff
@@ -17,6 +17,13 @@ import * as appReducer from 'state';
 
 import { initialState } from 'state/initialState';
 
+const createHistoryWithBasename = (historyOptions) => {
+  return useBasename(createHistory)({
+    basename: location.hostname === 'localhost' ? '/' : '/i-have-to-return-some-videotapes',
```

Comment:
Ah good point on `/editor`. But that route could be a constant or something and is always relative, so we could potentially do `location.pathname.replace('/editor', '')` which feels pretty nasty, but does keep everything relative and not hardcoded and not require any additional webpack :corn:fig.